### DISCUSSION TITLE
How to get debugging working when using @swc/jest?

### DISCUSSION THREAD

**Discussion by caveman-dick** [swc-project/swc#2769] [Q&A] [answered]
I have been trying for quite a bit today but I just can't figure out how to debug through my tests and code, is there something I need to do to enable debugging, or is it just not supported? I know debugging is working as I can stop on the SWC js code, just not on my typescript code.

Thanks

**Comment by hisomura** [comment, marked as answer]
You should use `souceMaps: "inline"` In .swcrc or jest.config.js 
https://swc.rs/docs/configuration/compilation#sourcemaps

**Reply by alexey-sh** [reply]
I received `Expected ';', '}' or <eof>` error with `{"sourceMaps": "inline"}`
So this does not work at all for me.

**Comment by SalvatorePreviti** [comment]
The problem seems to be happening only with .spec.ts file, or any file that contains a .something.ts
